pax-web-runtime/src/main/java/org/ops4j/pax/web/service/internal/ServletEventDispatcher.java

```diff
@@ -155,22 +148,6 @@ void destroy() {
 		}
 	}
 
-	private void sendInitialEvents(ServletListener listener) {
-		for (Map.Entry<Bundle, Map<String, ServletEvent>> entry : states
-				.entrySet()) {
-			try {
-				if (entry.getValue() != null && !entry.getValue().isEmpty()) {
-					for (ServletEvent event : entry.getValue().values()) {
-						callListener(listener, new ServletEvent(event, true));
-					}
-				}
-			} catch (RejectedExecutionException ree) {
-				LOG.warn("Executor shut down", ree);
-				break;
-			}
-		}
-	}
-
```

Comment:
Again, this functionality is intentional so if this is part of a memory leak we'll need to work around it but not remove it :-)